src/steps/push_branch_step.go

```diff
@@ -1,3 +1,4 @@
+//nolint:ireturn
```

Comment:
Why ignore for the entire file instead of just the specific function that returns an interface? Same for other steps

Reply:
Nevermind, appears fixed in #1714

Reply:
Yes, it later occurred to me that it's better to not ignore entire files, so I narrowed the linter exceptions to particular lines.